.cirrus.yml

```diff
@@ -27,6 +27,22 @@ stable_test_task:
     - cargo fmt --all -- --check
   before_cache_script: rm -rf $CARGO_HOME/registry/index
 
+no_std_test_task:
+  name: "Rust Stable (no_std)"
+  container:
+    image: rust:latest
+    cpu: 1
+    memory: 2Gi
+  cargo_cache:
+    folder: $CARGO_HOME/registry
+    fingerprint_script: cat Cargo.toml
+  setup_script:
+    - rustup target add thumbv6m-none-eabi
+  primary_test_script:
+    - rustc --version
+    - cargo build --no-default-features --target thumbv6m-none-eabi
```

Comment:
Completely unrelated, but thanks for showing me how to actually make some kind of test for a no-std crate. I had some [*really* nasty](https://github.com/shepmaster/twox-hash/blob/5c9a4690f576a20280a9ecbc4cbdc3ab2fff26e2/ci/check-no-std.sh#L25-L29) hacks to try and do this before.